src/shared/format-table.c

```diff
@@ -156,6 +156,9 @@ struct Table {
         size_t n_json_fields;
 
         bool *reverse_map;
+
+        size_t *delimiter_map; /* In vertical mode, after which rows a delimiter should be added. */
```

Comment:
why only in vertical mode btw? in regular systemctl outut we also draw horizontal lines, we should probably unify that...